dspace/modules/additions/src/main/java/org/dspace/discovery/FullTextContentStreams.java

```diff
@@ -94,7 +98,13 @@ private void buildFullTextList(Item parentItem) throws SQLException {
 
                     for (ResourcePolicy rp:bundlePolicies) {
                         if (rp.getdSpaceObject().getID() == fulltextBitstream.getID()) {
-                            if (rp.getGroup().getName().equalsIgnoreCase("anonymous")) {
+                            Date start = rp.getStartDate();
+                            Date end = rp.getEndDate();
+                            Date now = new Date();
+                            if (rp.getGroup().getName().equalsIgnoreCase("anonymous")
+                                && (start == null || ((start.before(now) || DateUtils.isSameDay(start, now))
+                                && (end == null || (now.after(end) || DateUtils.isSameDay(now, end)))))
```

Comment:
Am I reading this correctly?

If "now" is after "end", then...

The current date would be after the end date would result in a truth after the end date.

Should this instead be `now.before(end)`?

Reply:
Thanks. Will correct. FYI, no policy had an end date in our use case.

Reply:
I am going to go with `end.after(now)`.